data/json/recipes/recipe_ammo.json

```diff
@@ -2836,9 +2836,10 @@
     {"id":"CHEM","level":2,"amount":1}
   ], "tools": [
     [
-      [ "chemistry_set", 50 ],
       [ "hotplate", 50 ],
-      [ "toolset", 50 ]
+      [ "toolset", 50 ],
+      [ "fire", -1 ],
+      [ "char_smoker", 5 ]
```

Comment:
I'm seeing a LOT of these appear to remove the chem set from the tool charges list.  If that's because somehow the hotplate link lets folks drain charges that way, great, though IMO that's a bit fragile codewise.

More directly, it would have been very appreciated if you'd listed your changed recipes.  Black powder (which may or may not be DDA!"gunpowder", we've been over that a few times), sure, that's OK with primitive gear.